packages/mds-agency-cache/index.ts

```diff
@@ -100,7 +99,7 @@ async function getEventsInBBox(bbox: BoundingBox) {
   const events: string[] = await client.georadius(decorateKey('locations'), lng, lat, radius, 'm')
   const finish: Timestamp = now()
   const timeElapsed = finish - start
-  logger.info(`mds-agency-cache getEventsInBBox ${JSON.stringify(bbox)} time elapsed: ${timeElapsed}ms`)
+  AgencyCacheLogger.debug(`mds-agency-cache getEventsInBBox ${JSON.stringify(bbox)} time elapsed: ${timeElapsed}ms`)
```

Comment:
Having `mds-agency-cache` in the log text seems redundant now.

Reply:
Fixed in a7334b7